### PR TITLE
Generate random image and title

### PR DESCRIPTION
**Pull request by peytonjo**

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -89,4 +89,10 @@ var descriptors = [
...
   "blessings",
   "melancholy"
-];
\ No newline at end of file
+];
+
+module.exports = {
+  covers: covers,
+  titles: titles,
+  descriptors:descriptors
```
Comment (thaomonster) [inline src/data.js]: Lets add a space between the semicolon  and descriptors. Everything else looks good!

File: src/main.js [hunk 2]
```diff
@@ -17,3 +19,20 @@ var currentCover;
...
+  var randomImage = covers[randomImageIndex];
+
+  return randomImage;
+}
+
+
+//Get random titles
+function getRandomTitle(){
+  var randomTitleIndex = getRandomIndex(titles);
+  var randomTitle = tiles[randomTitleIndex];
```
Comment (thaomonster) [inline src/main.js]: There is a missing t in titles after the assign value.

File: src/data.js (hunk 1 above)
Comment (peytonjo) [reply src/data.js]: fixed

File: src/main.js (hunk 2 above)
Comment (peytonjo) [reply src/main.js]: fixed

**Pull request merged**